test-suite/tests/frame-g001-frame.jsonld

```diff
@@ -0,0 +1,13 @@
+{
+  "@context": {
+    "dc": "http://purl.org/dc/elements/1.1/",
+    "ex": "http://example.org/vocab#"
+  },
+  "@type": "ex:Library",
+  "ex:contains": {
+    "@type": "ex:Book",
+    "ex:contains": {
+      "@type": "ex:Chapter"
+    }
+  }
+}
```

Comment:
Rest of our test files all have an EOL so these should all as well.

Reply:
@dlongley Looking at other tests, most of them do not end with a newline, for example frame-0001-frame.jsonld. The only ones I find are the toRdf-*-out.nq files, as it's a required part of the N-Quads canonical form; it's not for JSON.

We could change these, but consistency is probably best and we should either touch every test file, and include some check in CI to verify, or leave as is.

Reply:
I vote for consistency whatever that is.